common-messaging/src/main/scala/org/genivi/sota/messaging/Messages.scala

```diff
@@ -36,6 +36,11 @@ object Messages {
     deviceId: Option[DeviceId],
     deviceType: DeviceType) extends BusMessage
 
+  final case class DeviceCredentialsSet(
```

Comment:
Why are we doing this? This credentials are like username/password and are floating around services like this, unencrypted, and are also stored unencrypted.